communication/src/AsyncTlsConnection.cpp

```diff
@@ -384,7 +382,7 @@ bool AsyncTlsConnection::verifyCertificateServer(bool preverified, boost::asio::
 
 std::pair<bool, NodeNum> AsyncTlsConnection::checkCertificate(X509* receivedCert,
                                                               std::string connectionType,
-                                                              std::string subject,
+                                                              const std::string& subject,
```

Comment:
I guess the changes in the CPP code are related to the updated in the clang, right?

Reply:
Yes this change is make clang-tidy happy :)